src/components/AvatarGroup/AvatarGroupCounter.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Tooltip from "../Tooltip/Tooltip";
+import Avatar from "../Avatar/Avatar";
+import PropTypes from "prop-types";
+import AvatarGroupCounterTooltipContainer from "./AvatarGroupCounterTooltipContainer";
+import cx from "classnames";
+import Counter from "../Counter/Counter";
+import styles from "./AvatarGroupCounter.module.scss";
+
+const AvatarGroupCounter = ({
+  counterTooltipAvatars,
+  counterProps,
+  counterTooltipCustomProps,
+  avatarSize,
+  avatarType
```

Comment:
why we need type in the level of the counter tooltip?

Reply:
Cause we are passing afterwards to avatars in tooltip, like this 
<Avatar {...avatarProps} ... type={type || avatarProps?.type} />
And we do the same for displaying avatars in AvatarGroup
This could be used to set type for all avatars at once - at AvatarGroup prop, not at every single avatar you are passing

But we can get rid of it, if u think its irrelevant